card/mockCard.go

```diff
@@ -652,18 +652,76 @@ func (c *MockCard) CreatePhonon(curveType model.CurveType) (keyIndex uint16, pub
 	return index, newp.PubKey, nil
 }
 
+func (c *MockCard) CreatePhononWithSetDescriptor(curveType model.CurveType, p *model.Phonon, privKey string) (keyIndex uint16, pubKey model.PhononPubKey, err error) {
```

Comment:
```suggestion
func (c *MockCard) CreatedBrandedPhonon(curveType model.CurveType, p *model.Phonon, privKey string) (keyIndex uint16, pubKey model.PhononPubKey, err error) {
```

Reply:
I dont think the `curveType` parameter is required as phonons created with this method won't be backed by crypto assets. Can created all branded phonons with the same default curve type.

Reply:
Will this method only be called to create branded or flexible phonons? If so, I would throw an error if the following coniditons arent met. 

len(privKey) > 0 || (p.CurrencyType == 4 || p.CurrencyType == 5)

And then move the public key generation out of the if block.